onnx/test/shape_inference_test.py

```diff
@@ -189,48 +238,94 @@ def _identity_prop(self, op: str, **kwargs: Any) -> None:
             graph, [make_tensor_value_info("y", TensorProto.FLOAT, (30, 4, 5))]
         )
 
+    def get_caller_function_name(self):
+        """Return the function name that calls this get_caller_function_name."""
```

Comment:
I’m not quite following this doc

Reply:
Forget that. This function is deleted in the new decorator design.